docs/getting-started/Installation.md

```diff
@@ -75,10 +75,16 @@ tool](https://docs.microsoft.com/en-us/ef/ef6/modeling/code-first/migrations/mig
 ### Command Line
 
 On the developer workstation, `migrate.exe` is in `packages\EntityFramework.6.2.0\tools`. You will want to
-copy it to the bin directory so that it is side-by-side with the DataAccess assembly. In the NuGet
+copy it to the Web.UI bin directory so that it is side-by-side with the DataAccess assembly. In the NuGet
 package described above, it is bundled into the `bin` directory. For detailed help, run `migrate.exe /?`.
-Here is a sample command. Note that you _must not_ have a `.\` or `./` at the beginning of the assembly
-name - doing so will cause an error.
+Here is a sample command.
+
+❗ You _must not_ have a `.\` or `./` at the beginning of the assembly name in
+the command below; doing so will cause an error.
+
+❗❗ This is a finicky old tool whose error messages are not always helpful. With
+the exact same source code, it may work on one computer and fail on another.
+Deploying from Visual Studio is a more consistent experience.
```

Comment:
This EF migrate command runs fine on the integration server, but it gets a bizarre error on the TeamCity build agent. Thus I had to leave the "create database" and "run unit test" steps disabled in TeamCity.